nexus/src/app/sagas/instance_update/mod.rs

```diff
@@ -1944,12 +1943,12 @@ mod test {
 
     // === migration source completed tests ===
 
-    #[nexus_test(server = crate::Server)]
+    #[nexus_test(server = crate::Server, extra_sled_agents = 1)]
     async fn test_migration_source_completed_succeeds(
         cptestctx: &ControlPlaneTestContext,
     ) {
         let _project_id = setup_test_project(&cptestctx.external_client).await;
-        let other_sleds = test_helpers::add_sleds(cptestctx, 1).await;
+        let other_sleds: Vec<_> = cptestctx.all_sled_agents().skip(1).collect();
```

Comment:
Nit, take it or leave it: we write this a _lot_...maybe `ControlPlaneTestContext` should just have a method that returns all of the extra sled agents?

Reply:
~~Also, perhaps we could have the `ControlPlaneTextContext` return that as a slice, using `&self.sled_agents[1..]`, so that callers don't need to always iterate over the sled agents and collect them?~~ 

**Edit**: Oh, disregard that last bit, I see that `all_ssled_agents` is returning the `sa.server()`, so it does have to be an iterator. Carry on.

Reply:
Done in b7de7c601, thanks :)